### PR TITLE
Fix undo

### PR DESCRIPTION
**Pull request by rossxhunter**

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import undoable, { excludeAction } from 'redux-undo'
+import undoable, { excludeAction, groupByActionTypes } from 'redux-undo'
```
Comment (jasonlipo) [inline src/reducers/index.js]: Are you using `groupByActionTypes`?

File: src/reducers/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 import iprove from './iprove'
-import { ADD_IDENTIFIERS, ADD_ATOMS, ADD_RELATIONS } from '../constants';
+import { ADD_IDENTIFIERS, ADD_ATOMS, ADD_RELATIONS, ADD_TYPES, UPDATE_RULE, SET_SCOPE, NEW_STEP } from '../constants';
```
Comment (azakon97) [inline src/reducers/index.js]: UPDATE_RULE,  SET_SCOPE, NEW_STEP are not being used

File: src/reducers/index.js (hunk 1 above)
Comment (azakon97) [inline src/reducers/index.js]: groupByActionTypes is not being used anywhere

Comment (rossxhunter) [reply src/reducers/index.js]: no, I'll remove

File: src/reducers/index.js (hunk 2 above)
Comment (rossxhunter) [reply src/reducers/index.js]: Have removed them now

File: src/reducers/index.js (hunk 1 above)
Comment (rossxhunter) [reply src/reducers/index.js]: Removed

**Pull request merged**